docs/source/contributor-guide/index.md

```diff
@@ -19,17 +19,40 @@
 
 # Introduction
 
-We welcome and encourage contributions of all kinds, such as:
+We welcome and encourage contributions of all kinds, from all levels, such as:
 
-1. Tickets with issue reports of feature requests
-2. Documentation improvements
-3. Code, both PR and (especially) PR Review.
+1. Tickets with issue reports or feature requests
+2. Discussions
+3. Documentation improvements
+4. Code, both PR and (especially) PR Review.
 
-In addition to submitting new PRs, we have a healthy tradition of community members reviewing each other's PRs. Doing so is a great way to help the community as well as get more familiar with Rust and the relevant codebases.
+In addition to submitting new PRs, we have a healthy tradition of community
+members reviewing each other's PRs. Doing so is a great way to help the
+community as well as get more familiar with Rust and the relevant codebases.
+
+## Finding and Creating Issues to Work On
+
+You can find a curated [good-first-issue] list to help you get started.
```

Comment:
```suggestion
You can find a curated [good-first-issue](https://github.com/apache/arrow-datafusion/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) list to help you get started.
```


How about moving the link over here? It previously displayed as:
> <img width="400" alt="截屏2024-01-27 11 57 53" src="https://github.com/apache/arrow-datafusion/assets/26053282/061499e6-5bbd-4a75-8e44-baa702716a1d">

Reply:
Good call -- thank you @jonahgao